src/imitation/util/logger.py

```diff
@@ -103,27 +165,86 @@ def _update_name_to_maps(self) -> None:
         self.name_to_excluded = self._logger.name_to_excluded
 
     @contextlib.contextmanager
-    def accumulate_means(self, subdir: types.AnyPath) -> Generator[None, None, None]:
+    def add_prefix(self, prefix: str) -> Generator[None, None, None]:
```

Comment:
Now there are multiple prefixes calling this just `add_prefix` feels a bit ambiguous, consider renaming? E.g. `add_accumulate_prefix` or `add_subdir_prefix`?